server/db/seed.js

```diff
@@ -0,0 +1,61 @@
+var db = require('./db.js');
+
+var seed = function(n) {
+
+  var genRandomInt = function(min, max) {
+    return Math.floor(Math.random() * (max - min + 1) + min);
+  };
+
+  var props = [];
+  var cals = [];
+
+  for (var i = 1; i <= n; i++) {
```

Comment:
I'd suggest using let instead of var in for loops, generally (especially since you use let in the next for loop)